src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,5 @@
+fun main() {
+    println("Enter your city…")
+    val city = readLine()
+    println("User lives in $city")
+}
```

Comment:
we should have blank line at the EOF
```suggestion
}

```